incubator/spark-history-server/templates/rbac.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
```

Comment:
All of the community supported versions of Kubernetes suppot the API version `rbac.authorization.k8s.io/v1`. Can we use that for the RBAC usage here?